frontend/components/card/search-result/CardSearchResultResource.vue

```diff
@@ -43,9 +43,7 @@
           <MetaTagLocation :location="resource?.relatedLocation" />
           <MetaTagDate :date="resource.creationDate.toLocaleDateString()" />
         </div>
-        <div
-          class="flex justify-center space-x-3 lg:space-x-4 md:justify-start"
-        >
+        <div class="flex justify-center space-x-3 lg:space-x-4 md:justify-start">
```

Comment:
Something seems to be wrong with this here as it's not formatted as a string. Is problematic before, so just wanted to bring it up so we can fix it 😊

Reply:
Nvm, I guess GitHub formatting is hallucinating 🤷‍♂️